<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::PreferencesDialog</class>
 <widget class="QDialog" name="tt3::gui::PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-gui.qrc">
    <normaloff>:/tt3-gui/Resources/Images/Actions/PreferencesLarge.png</normaloff>:/tt3-gui/Resources/Images/Actions/PreferencesLarge.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTreeWidget" name="preferencesTreeWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>200</horstretch>
       <verstretch>200</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="editorsFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="resetPushButton">
     <property name="text">
      <string>Reset all</string>
     </property>
     <property name="icon">
      <iconset resource="tt3-gui.qrc">
       <normaloff>:/tt3-gui/Resources/Images/Actions/ResetSmall.png</normaloff>:/tt3-gui/Resources/Images/Actions/ResetSmall.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>93</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::gui::PreferencesDialog</receiver>
   <slot>_accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::gui::PreferencesDialog</receiver>
   <slot>_reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preferencesTreeWidget</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>tt3::gui::PreferencesDialog</receiver>
   <slot>_preferencesTreeWidgetCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::gui::PreferencesDialog</receiver>
   <slot>_resetPushNuttonClocked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_preferencesTreeWidgetCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>_accept()</slot>
  <slot>_reject()</slot>
  <slot>_resetPushNuttonClocked()</slot>
 </slots>
</ui>
